web/server/vue-cli/src/views/Reports.vue

```diff
@@ -111,7 +111,7 @@
             :to="{ name: 'report-detail', query: {
               ...$router.currentRoute.query,
               'report-id': item.reportId ? item.reportId : undefined,
-              'report-hash': item.reportId ? undefined : item.bugHash
+              'report-hash': item.bugHash
```

Comment:
Please also add the file name in the url of the reques to show the bug in the correct file in case the report id is missing.